### PR TITLE
REPL option to see claim

### LINKED ISSUE #1657: Add REPL option to see claim RHS
Will help debug final implications from REPL.

### PR DESCRIPTION
**Pull request by andreiburdusa**
---
Fixes #1657 
Please see my comment below
###### Reviewer checklist

- [ ] Test coverage: `stack test --coverage`
- [ ] Public API documentation: `stack haddock`

### CODE REVIEW

**Comment by andreiburdusa** [comment]
I'm thinking about replacing

> The commands which output K have similar names to their KORE outputting counterparts, e.g. `konfig` instead of `config` or `ktry` instead of `try.` See the `kast.kscript` for the full list of K commands.

with

> The commands which output K have similar names to their KORE outputting counterparts, e.g. `konfig` instead of `config` or `ktry` instead of `try`. Some can be sufixed with `-n`, `-d` or `-nd` for specifying the argument, the directory or both, respectively. See the `kast.kscript` for the full list of K commands.

File: dist/kast.kscript [hunk 1]
```diff
@@ -1,5 +1,7 @@
-alias kclaim x = claim x | kast -i kore -o pretty -d . /dev/stdin
-alias kclaim-d x y = claim x | kast -i kore -o pretty -d y /dev/stdin
+alias kclaim = claim | kast -i kore -o pretty -d . /dev/stdin
```
Comment (andreiburdusa) [inline dist/kast.kscript]: Actually, I noticed this does not work, but it did work before either:
on branch master I get the following error:
```Kore (0)> kclaim 0
[Error] Critical: Parse error
 (ParseError: ERROR: Line 3: Column 5: Expected '(', but '\'
    \implies{SortGeneratedTopCell{}}(
    ^)```

Comment (ttuegel) [reply dist/kast.kscript]: What does `claim 0` print?

**Comment by ttuegel** [comment]
> Some can be suffixed with `-n`, `-d` or `-nd` for specifying the argument, the directory or both, respectively.

This is good; it would be better to say exactly _which_ commands accept those options.

File: kore/src/Kore/Repl/Data.hs [hunk 2]
```diff
@@ -290,7 +290,8 @@ helpText =
     \help                                     shows this help message\n\
     \claim [n|<name>]                         shows the nth claim, the claim with\
                                               \ <name> or if used without args\
-                                              \ shows the currently focused claim\n\
+                                              \ shows the currently focused claim\
+                                              \ in the form: LHS => (Operator) RHS \n\
```
Comment (ttuegel) [inline kore/src/Kore/Repl/Data.hs]: Would "modality" be more specific than "operator"? Or is "modality" not quite the right word?

Comment (andreiburdusa) [reply kore/src/Kore/Repl/Data.hs]: I think it is

File: dist/kast.kscript (hunk 1 above)
Comment (andreiburdusa) [reply dist/kast.kscript]:
```
Kore (0)> claim 0
claim {}
/* Spa */
    \implies{SortGeneratedTopCell{}}(
        /* Spa */
        \and{SortGeneratedTopCell{}}(
            /* D Sfa */ \top{SortGeneratedTopCell{}}(),
            /* Fl Fn D Spa */
            Lbl'-LT-'generatedTop'-GT-'{}(
                /* Fl Fn D Spa */
                Lbl'-LT-'T'-GT-'{}(
                    /* Fl Fn D Sfa Cl */
                    Lbl'-LT-'k'-GT-'{}(
                        /* Fl Fn D Sfa Cl */
                        kseq{}(
                            /* Fl Fn D Sfa Cli */
                            /* Inj: */ inj{SortStmt{}, SortKItem{}}(
                                /* Fl Fn D Sfa Cl */
                                Lblif'LParUndsRParUnds'else'UndsUnds'IMP-SYNTAX'Unds'Stmt'Unds'BExp'Unds'Block'Unds'Block{}(
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'Unds-LT-EqlsUndsUnds'IMP-SYNTAX'Unds'BExp'Unds'AExp'Unds'AExp{}(
                                        /* Fl Fn D Sfa Cli */
                                        /* Inj: */ inj{SortId{}, SortAExp{}}(
                                            /* Fl Fn D Sfa Cl */
                                            \dv{SortId{}}(/* Fl Fn D Sfa Cl */
                                            "a")
                                        ),
                                        /* Fl Fn D Sfa Cli */
                                        /* Inj: */ inj{SortId{}, SortAExp{}}(
                                            /* Fl Fn D Sfa Cl */
                                            \dv{SortId{}}(/* Fl Fn D Sfa Cl */
                                            "b")
                                        )
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'LBraUndsRBraUnds'IMP-SYNTAX'Unds'Block'Unds'Stmt{}(
                                        /* Fl Fn D Sfa Cl */
                                        Lbl'UndsEqlsUndsSClnUnds'IMP-SYNTAX'Unds'Stmt'Unds'Id'Unds'AExp{}(
                                            /* Fl Fn D Sfa Cl */
                                            \dv{SortId{}}(/* Fl Fn D Sfa Cl */
                                            "max"),
                                            /* Fl Fn D Sfa Cli */
                                            /* Inj: */ inj{SortId{}, SortAExp{}}(
                                                /* Fl Fn D Sfa Cl */
                                                \dv{SortId{}}(/* Fl Fn D Sfa Cl */
                                                "b")
                                            )
                                        )
                                    ),
                                    /* Fl Fn D Sfa Cl */
                                    Lbl'LBraUndsRBraUnds'IMP-SYNTAX'Unds'Block'Unds'Stmt{}(
                                        /* Fl Fn D Sfa Cl */
                                        Lbl'UndsEqlsUndsSClnUnds'IMP-SYNTAX'Unds'Stmt'Unds'Id'Unds'AExp{}(
                                            /* Fl Fn D Sfa Cl */
                                            \dv{SortId{}}(/* Fl Fn D Sfa Cl */
                                            "max"),
                                            /* Fl Fn D Sfa Cli */
                                            /* Inj: */ inj{SortId{}, SortAExp{}}(
                                                /* Fl Fn D Sfa Cl */
                                                \dv{SortId{}}(/* Fl Fn D Sfa Cl */
                                                "a")
                                            )
                                        )
                                    )
                                )
                            ),
                            /* Fl Fn D Sfa Cl */ dotk{}()
                        )
                    ),
                    /* Fl Fn D Spa */
                    Lbl'-LT-'state'-GT-'{}(
                        /* Fl Fn D Spa */
                        /* builtin: */
                        Lbl'Unds'Map'Unds'{}(
                            /* concrete element: */ Lbl'UndsPipe'-'-GT-Unds'{}(
                                /* Fl Fn D Sfa Cli */
                                /* Inj: */ inj{SortId{}, SortKItem{}}(
                                    /* Fl Fn D Sfa Cl */
                                    \dv{SortId{}}(/* Fl Fn D Sfa Cl */ "a")
                                ),
                                /* Fl Fn D Spa */
                                /* Inj: */ inj{SortInt{}, SortKItem{}}(
                                    /* Fl Fn D Sfa */ VarA:SortInt{}
                                )
                            ),
                        Lbl'Unds'Map'Unds'{}(
                            /* concrete element: */ Lbl'UndsPipe'-'-GT-Unds'{}(
                                /* Fl Fn D Sfa Cli */
                                /* Inj: */ inj{SortId{}, SortKItem{}}(
                                    /* Fl Fn D Sfa Cl */
                                    \dv{SortId{}}(/* Fl Fn D Sfa Cl */ "b")
                                ),
                                /* Fl Fn D Spa */
                                /* Inj: */ inj{SortInt{}, SortKItem{}}(
                                    /* Fl Fn D Sfa */ VarB:SortInt{}
                                )
                            ),
                            /* concrete element: */ Lbl'UndsPipe'-'-GT-Unds'{}(
                                /* Fl Fn D Sfa Cli */
                                /* Inj: */ inj{SortId{}, SortKItem{}}(
                                    /* Fl Fn D Sfa Cl */
                                    \dv{SortId{}}(/* Fl Fn D Sfa Cl */ "max")
                                ),
                                /* Fl Fn D Sfa */ Var'Unds'0:SortKItem{}
                            )
                        ))
                    )
                ),
                /* Fl Fn D Spa */
                Lbl'-LT-'generatedCounter'-GT-'{}(
                    /* Fl Fn D Sfa */ VarDotVar00:SortInt{}
                )
            )
        ),
        /* Spa */
        weakAlwaysFinally{SortGeneratedTopCell{}}(
            /* Spa */
            \exists{SortGeneratedTopCell{}}(
                Var'Ques'M:SortInt{},
                /* Spa */
                \and{SortGeneratedTopCell{}}(
                    /* Spa */
                    \equals{SortBool{}, SortGeneratedTopCell{}}(
                        /* Fl Fn D Spa */
                        Lbl'Unds'orBool'UndsUnds'BOOL'Unds'Bool'Unds'Bool'Unds'Bool{}(
                            /* Fl Fn D Spa */
                            Lbl'Unds'andBool'Unds'{}(
                                /* Fl Fn D Spa */
                                Lbl'Unds-GT-Eqls'Int'Unds'{}(
                                    /* Fl Fn D Sfa */ VarA:SortInt{},
                                    /* Fl Fn D Sfa */ VarB:SortInt{}
                                ),
                                /* Fl Fn D Spa */
                                Lbl'UndsEqlsEqls'Int'Unds'{}(
                                    /* Fl Fn D Sfa */ Var'Ques'M:SortInt{},
                                    /* Fl Fn D Sfa */ VarA:SortInt{}
                                )
                            ),
                            /* Fl Fn D Spa */
                            Lbl'Unds'andBool'Unds'{}(
                                /* Fl Fn D Spa */
                                Lbl'Unds-GT-Eqls'Int'Unds'{}(
                                    /* Fl Fn D Sfa */ VarB:SortInt{},
                                    /* Fl Fn D Sfa */ VarA:SortInt{}
                                ),
                                /* Fl Fn D Spa */
                                Lbl'UndsEqlsEqls'Int'Unds'{}(
                                    /* Fl Fn D Sfa */ Var'Ques'M:SortInt{},
                                    /* Fl Fn D Sfa */ VarB:SortInt{}
                                )
                            )
                        ),
                        /* Fl Fn D Sfa Cl */
                        /* builtin: */ \dv{SortBool{}}("true")
                    ),
                    /* Fn Spa */
                    Lbl'-LT-'generatedTop'-GT-'{}(
                        /* Fn Spa */
                        Lbl'-LT-'T'-GT-'{}(
                            /* Fl Fn D Sfa Cl */
                            Lbl'-LT-'k'-GT-'{}(/* Fl Fn D Sfa Cl */ dotk{}()),
                            /* Fn Spa */
                            Lbl'-LT-'state'-GT-'{}(
                                /* Fn Spa */
                                Lbl'Unds'Map'Unds'{}(
                                    /* Fn Spa */
                                    Lbl'Unds'Map'Unds'{}(
                                        /* Fl Fn D Spa */
                                        Lbl'UndsPipe'-'-GT-Unds'{}(
                                            /* Fl Fn D Sfa Cli */
                                            /* Inj: */ inj{SortId{}, SortKItem{}}(
                                                /* Fl Fn D Sfa Cl */
                                                \dv{SortId{}}(/* Fl Fn D Sfa Cl */
                                                "a")
                                            ),
                                            /* Fl Fn D Spa */
                                            /* Inj: */ inj{SortInt{}, SortKItem{}}(
                                                /* Fl Fn D Sfa */ VarA:SortInt{}
                                            )
                                        ),
                                        /* Fl Fn D Spa */
                                        Lbl'UndsPipe'-'-GT-Unds'{}(
                                            /* Fl Fn D Sfa Cli */
                                            /* Inj: */ inj{SortId{}, SortKItem{}}(
                                                /* Fl Fn D Sfa Cl */
                                                \dv{SortId{}}(/* Fl Fn D Sfa Cl */
                                                "b")
                                            ),
                                            /* Fl Fn D Spa */
                                            /* Inj: */ inj{SortInt{}, SortKItem{}}(
                                                /* Fl Fn D Sfa */ VarB:SortInt{}
                                            )
                                        )
                                    ),
                                    /* Fl Fn D Spa */
                                    Lbl'UndsPipe'-'-GT-Unds'{}(
                                        /* Fl Fn D Sfa Cli */
                                        /* Inj: */ inj{SortId{}, SortKItem{}}(
                                            /* Fl Fn D Sfa Cl */
                                            \dv{SortId{}}(/* Fl Fn D Sfa Cl */
                                            "max")
                                        ),
                                        /* Fl Fn D Spa */
                                        /* Inj: */ inj{SortInt{}, SortKItem{}}(
                                            /* Fl Fn D Sfa */
                                            Var'Ques'M:SortInt{}
                                        )
                                    )
                                )
                            )
                        ),
                        /* Fl Fn D Spa */
                        Lbl'-LT-'generatedCounter'-GT-'{}(
                            /* Fl Fn D Sfa */ VarDotVar00:SortInt{}
                        )
                    )
                )
            )
        )
    )
[]
```

**Comment by andreiburdusa** [comment]
> > Some can be suffixed with `-n`, `-d` or `-nd` for specifying the argument, the directory or both, respectively.
> 
> This is good; it would be better to say exactly _which_ commands accept those options.

how about

> The commands which output K have similar names to their KORE outputting counterparts, e.g. `konfig` instead of `config` or `ktry` instead of `try`. Some can be suffixed with `-n`, `-d` or `-nd` for specifying the argument, the directory or both, respectively.
`kcliam`, `konfig`, `krule` : don't require the argument for their counterparts; can be suffixed with `-n`, `-d` or `-nd`
`kaxiom`, `ktry`, `ktryf` : require the argument for their counterparts; can be suffixed with `-d`
See the `kast.kscript` for the full list of K commands.

**Comment by ttuegel** [comment]
Sounds good!

**Pull request merged**